<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArrangeWindow</class>
 <widget class="QMainWindow" name="ArrangeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBoxSort">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #ddd;
    border-radius: 5px;
    margin-top: 10px;
    font-size: 16px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
    background-color: #F0F0F0;
}

QGroupBox::indicator {
    width: 18px;
    height: 18px;
    border: 1px solid #ddd;
    border-radius: 9px;
}

QGroupBox::indicator:checked {
    background-color: #2196f3;
    border-color: #2196f3;
}
</string>
        </property>
        <property name="title">
         <string>排序</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>排序字段</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="rBtnDescend">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
    font-size: 14px;
}

QRadioButton::indicator {
    width: 8px;
    height: 8px;
    border-radius: 50%;
    border: 2px solid #C0C0C0;
}

QRadioButton::indicator:checked {
    border-color: #0078D7;
    background-color: #0078D7;
}

QRadioButton::indicator:hover {
    border-color: #0078D7;
}

QRadioButton::indicator:checked:hover {
    background-color: #0078D7;
}
</string>
           </property>
           <property name="text">
            <string>降序</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="rBtnAscend">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
    font-size: 14px;
}

QRadioButton::indicator {
    width: 8px;
    height: 8px;
    border-radius: 50%;
    border: 2px solid #C0C0C0;
}

QRadioButton::indicator:checked {
    border-color: #0078D7;
    background-color: #0078D7;
}

QRadioButton::indicator:hover {
    border-color: #0078D7;
}

QRadioButton::indicator:checked:hover {
    background-color: #0078D7;
}
</string>
           </property>
           <property name="text">
            <string>升序</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="comboFilter"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBoxSearch">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #ddd;
    border-radius: 5px;
    margin-top: 10px;
    font-size: 16px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
    background-color: #F0F0F0;
}

QGroupBox::indicator {
    width: 18px;
    height: 18px;
    border: 1px solid #ddd;
    border-radius: 9px;
}

QGroupBox::indicator:checked {
    background-color: #2196f3;
    border-color: #2196f3;
}
</string>
        </property>
        <property name="title">
         <string>检索</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QPushButton" name="btnSearch">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #3385ff;
    border: none;
    color: #fff;
    padding: 3px 10px;
    border-radius: 3px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #2d78f4;
}
</string>
           </property>
           <property name="text">
            <string>项目搜索</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="editEventsID"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBoxFilter">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid #ddd;
    border-radius: 5px;
    margin-top: 10px;
    font-size: 16px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
    padding: 0 5px;
    background-color: #F0F0F0;
}

QGroupBox::indicator {
    width: 18px;
    height: 18px;
    border: 1px solid #ddd;
    border-radius: 9px;
}

QGroupBox::indicator:checked {
    background-color: #2196f3;
    border-color: #2196f3;
}
</string>
        </property>
        <property name="title">
         <string>过滤</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QRadioButton" name="rBtnBoth">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
    font-size: 14px;
}

QRadioButton::indicator {
    width: 8px;
    height: 8px;
    border-radius: 50%;
    border: 2px solid #C0C0C0;
}

QRadioButton::indicator:checked {
    border-color: #0078D7;
    background-color: #0078D7;
}

QRadioButton::indicator:hover {
    border-color: #0078D7;
}

QRadioButton::indicator:checked:hover {
    background-color: #0078D7;
}
</string>
           </property>
           <property name="text">
            <string>全显示</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="rBtnR">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
    font-size: 14px;
}

QRadioButton::indicator {
    width: 8px;
    height: 8px;
    border-radius: 50%;
    border: 2px solid #C0C0C0;
}

QRadioButton::indicator:checked {
    border-color: #0078D7;
    background-color: #0078D7;
}

QRadioButton::indicator:hover {
    border-color: #0078D7;
}

QRadioButton::indicator:checked:hover {
    background-color: #0078D7;
}
</string>
           </property>
           <property name="text">
            <string>复赛</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="rBtnP">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
    font-size: 14px;
}

QRadioButton::indicator {
    width: 8px;
    height: 8px;
    border-radius: 50%;
    border: 2px solid #C0C0C0;
}

QRadioButton::indicator:checked {
    border-color: #0078D7;
    background-color: #0078D7;
}

QRadioButton::indicator:hover {
    border-color: #0078D7;
}

QRadioButton::indicator:checked:hover {
    background-color: #0078D7;
}
</string>
           </property>
           <property name="text">
            <string>初赛</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="rBtnF">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
    font-size: 14px;
}

QRadioButton::indicator {
    width: 8px;
    height: 8px;
    border-radius: 50%;
    border: 2px solid #C0C0C0;
}

QRadioButton::indicator:checked {
    border-color: #0078D7;
    background-color: #0078D7;
}

QRadioButton::indicator:hover {
    border-color: #0078D7;
}

QRadioButton::indicator:checked:hover {
    background-color: #0078D7;
}
</string>
           </property>
           <property name="text">
            <string>决赛</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddArrange"/>
   <addaction name="actModifyArrange"/>
   <addaction name="actDeleteArrange"/>
   <addaction name="actExportArrange"/>
   <addaction name="separator"/>
   <addaction name="actClose"/>
  </widget>
  <action name="actClose">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/images/706.bmp</normaloff>:/icons/images/706.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
  </action>
  <action name="actDeleteArrange">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/images/324.bmp</normaloff>:/icons/images/324.bmp</iconset>
   </property>
   <property name="text">
    <string>删除赛程</string>
   </property>
   <property name="toolTip">
    <string>删除赛程</string>
   </property>
  </action>
  <action name="actModifyArrange">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/images/124.bmp</normaloff>:/icons/images/124.bmp</iconset>
   </property>
   <property name="text">
    <string>修改赛程</string>
   </property>
   <property name="toolTip">
    <string>修改赛程安排</string>
   </property>
  </action>
  <action name="actAddArrange">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/images/316.bmp</normaloff>:/icons/images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>添加赛程</string>
   </property>
   <property name="toolTip">
    <string>添加赛程安排</string>
   </property>
  </action>
  <action name="actExportArrange">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/images/Excel.ico</normaloff>:/icons/images/Excel.ico</iconset>
   </property>
   <property name="text">
    <string>导出信息</string>
   </property>
   <property name="toolTip">
    <string>导出赛程安排信息</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actClose</sender>
   <signal>triggered()</signal>
   <receiver>ArrangeWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
